utils/token.ts

```diff
@@ -1,7 +1,8 @@
 import { GNOSIS_SCAN_URL } from '@/constants';
 
+export const getExplorerUrl = (txHash: string, isTx: boolean = true) => {
```

Comment:
Instead of using a boolean, lets have 2 fuctions, one for transactions and other for addresses. That way we know what it does by the name.